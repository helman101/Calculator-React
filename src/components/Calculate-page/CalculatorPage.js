import React from 'react';
import Calculator from '../calculator/Calculator';
import * as styles from '../../style.module.css';

const CalculatorPage = () => (
  <div className={`${styles.pages} ${styles.dFlex} ${styles.width100} ${styles.justifySpaceBetween}`}>
    <h1>Letâ€™s do some math!</h1>
    <Calculator />
  </div>
);

export default CalculatorPage;
