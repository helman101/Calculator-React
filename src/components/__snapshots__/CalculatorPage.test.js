import React from 'react';
import renderer from 'react-test-renderer';
import CalculatorPage from '../Calculate-page/CalculatorPage';

it('renders correctly', () => {
  const calculatorPage = renderer
    .create(<CalculatorPage />)
    .toJSON();
  expect(calculatorPage).toMatchSnapshot();
});
