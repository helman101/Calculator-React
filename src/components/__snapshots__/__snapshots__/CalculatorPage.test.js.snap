// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="undefined undefined undefined undefined"
>
  <h1>
    Let’s do some math!
  </h1>
  <div
    className="undefined undefined undefined undefined"
  >
    <div
      className="undefined undefined undefined"
    >
      <p>
        0
      </p>
    </div>
    <div
      className="undefined undefined"
    >
      <div
        className="undefined"
      >
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="AC"
        >
          AC
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="+/-"
        >
          +/-
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="%"
        >
          %
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="÷"
        >
          ÷
        </button>
      </div>
      <div
        className="undefined"
      >
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="7"
        >
          7
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="8"
        >
          8
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="9"
        >
          9
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="X"
        >
          X
        </button>
      </div>
      <div
        className="undefined"
      >
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="4"
        >
          4
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="5"
        >
          5
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="6"
        >
          6
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="-"
        >
          -
        </button>
      </div>
      <div
        className="undefined"
      >
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="1"
        >
          1
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="2"
        >
          2
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="3"
        >
          3
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="+"
        >
          +
        </button>
      </div>
      <div
        className="undefined"
      >
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="0"
        >
          0
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="."
        >
          .
        </button>
        <button
          className="undefined undefined undefined"
          onClick={[Function]}
          type="button"
          value="="
        >
          =
        </button>
      </div>
    </div>
  </div>
</div>
`;
